tools/zh_segmenter/segment.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+import argparse
+import pathlib
+import os
+import re
+import sys
+
+import tensorflow_text
+
+_DISALLOW_SEGMENTATION = re.compile(r'(?:\{[^\}]*\}|\\[a-z]|\s)+'.encode())
+
+_ZWSP = "\u200B"
```

Comment:
Should we strip ZWSP before running the model, or is it able to cope with previously inserted break points?

Reply:
We do this in `_RemoveSegmentationMarkers`.